src/groupByKey.ts

```diff
@@ -1,7 +1,24 @@
+
 type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  let obj: GroupsMap<T> = {};
```

Comment:
I think you can change name something like grouped or so on. And what about let?

Reply:
oops, you are right!) let because in forEach method I change this object